src/moonbeam/moonbase-local.js

```diff
@@ -293,12 +294,22 @@ const main = async () => {
         feePerSecond,
         keyPair: aliceKeyPair,
     });
-    const taskId = await turingHelper.api.rpc.automationTime.generateTaskId(turingAddress, providedId);
+    const taskId = await turingHelper.api.rpc.automationTime.generateTaskId(proxyOnTuring, providedId);
+
+    // Check that the task has been successfully added to the task list
+    console.log('\n5. Check that the task has been successfully added to the task list ...');
+    if (schedule.Fixed?.executionTimes[0] !== 0) {
+        console.log('\nWait for 1 minute for the execution of the XCM message to schedule task on Turing ...');
+        await delay(60000);
```

Comment:
The console.log `Wait for 1 minute for the execution of the XCM message to schedule task on Turing ...` is wrong.

This line is not check for execution, but task addition. Please fix the console.log.

Also, waiting for 20 seconds should be more than enough. Don’t wait for 1 minute.